MA/Paper: Display a warning when changing datum values

Jnnybrrn opened the pull request:
When the user focuses on a datum which is not empty, change
the class of the changewarning field to re-trigger the animation.

The warning will pulse once from black -> red -> black.

In addition, populate this warning box on page load ONLY if the
datum is not empty, to prevent any wierd popping from focus/unfocus.

Commit:
MA/Paper: Display a warning when changing datum values

When the user focuses on a datum which is not empty, change
the class of the changewarning field to re-trigger the animation.

The warning will pulse once from black -> red -> black.

In addition, populate this warning box on page load ONLY if the
datum is not empty, to prevent any wierd popping from focus/unfocus.

Jnnybrrn commented:
Trello card - https://trello.com/b/WDTttp0z/lima

jacekkopecky commented on webpages/js/metaanalyses.js:
```diff
@@ -1416,10 +1416,11 @@
               td.classList.add('empty');
             } else {
               _.fillEls(td, '.value', val.value);
+              _.fillEls(td, '.changewarning', "When updating an existing value, consider leaving a comment justifying the change.")
```
please move the content into the HTML

jacekkopecky commented on webpages/js/metaanalyses.js:
```diff
@@ -1429,6 +1430,16 @@
             _.setProps(td, '.valenteredby', 'href', '/' + (val && val.enteredBy || user) + '/');
             _.fillEls (td, '.valctime', _.formatDateTime(val && val.ctime || Date.now()));
 
+            // Set up a listener to onfocus trigger the animation on the warning element
```
this may be replaced in CSS by
```
.value.editing:focus + .datum.popupbox .changewarning { animation: ... }
```
not sure about `:focus`

jacekkopecky commented on webpages/profile/metaanalysis.html:
```diff
@@ -14,7 +14,7 @@
     <h1>LiMA</h1><h2>Living Meta-Analysis</h2>
   </a>
-  
+
```
fixme

jacekkopecky reviewed (changes requested):
small tweaks required

Commit:
MA/Papers: Update handling of changewarning

Now the animation is driven entirely by CSS

No longer using class 'animated', simply animate on focus of
contenteditable sibling.

Changes here in response to review comments.

Jnnybrrn commented:
New commit w/ feedback changes, ready for review
